Rx/v2/src/rxcpp/rx-observable.hpp

```diff
@@ -723,6 +723,47 @@ class observable
         return detail_subscribe(make_subscriber<T>(std::forward<ArgN>(an)...));
     }
 
+    /*! Returns an Observable that emits true if any item emitted by the source Observable satisfies a specified condition, otherwise false.
+        Emits false if the source Observable terminates without emitting any item.
+
+        \tparam Predicate  the type of the test function.
+
+        \param p  the test function to test items emitted by the source Observable.
+
+        \return  Observable that emits true if any item emitted by the source observable satisfies a specified condition, otherwise false.
+
+        \sample
+        \snippet exists.cpp exists sample
+        \snippet output.txt exists sample
+    */
+    template<class Predicate>
+    auto exists(Predicate p) const
+        /// \cond SHOW_SERVICE_MEMBERS
+        -> decltype(EXPLICIT_THIS lift<bool>(rxo::detail::any<T, Predicate>(std::move(p))))
+        /// \endcond
+    {
+        return                    lift<bool>(rxo::detail::any<T, Predicate>(std::move(p)));
+    }
+
+    /*! Returns an Observable that emits true if the source Observable emitted a specified item, otherwise false.
+        Emits false if the source Observable terminates without emitting any item.
+
+        \param value  the item to search for.
+
+        \return  Observable that emits true if the source Observable emitted a specified item, otherwise false.
+
+        \sample
+        \snippet contains.cpp contains sample
+        \snippet output.txt contains sample
+    */
+    auto contains(T value) const
+        /// \cond SHOW_SERVICE_MEMBERS
+        -> decltype(EXPLICIT_THIS lift<bool>(rxo::detail::any<T, std::function<bool(T)>>(std::function<bool(T)>{})))
```

Comment:
I would like to avoid function usage since it adds a virtual call per value and a potential allocation (and there for a throw..)

However, that does not block this PR.

Thanks!